sematic/versions.py

```diff
@@ -23,7 +23,7 @@
 # at the CURRENT_VERSION. Should be updated any time a breaking change
 # is made to the web API. If there is a breaking change, there should
 # be a TODO below
-MIN_CLIENT_SERVER_SUPPORTS = (0, 24, 1)
+MIN_CLIENT_SERVER_SUPPORTS = (0, 30, 0)
```

Comment:
I believe the first entry `TO DEPRECATE` on line 16 can be removed now. Please check that this isn't the Dashboard typing bug you fixed by re-instating the code.
The entries from lines 17 and 19 should be moved to `0.32.0`, because we haven't paid them any attention.

Reply:
done